src/workerd/io/BUILD.bazel

```diff
@@ -351,13 +352,17 @@ kj_test(
 
 kj_test(
     src = "promise-wrapper-test.c++",
-    deps = [":io"],
+    deps = [
+        ":io",
+        "//src/workerd/api:urlpattern",
```

Comment:
Why does promise-wrapper-test depend on jsgpattern?

Reply:
I have no idea. I am investigating this at the moment.